src/Umbraco.Core/Models/UserExtensions.cs

```diff
@@ -219,14 +219,14 @@ private static bool StartsWithPath(string test, string path)
 
         private static string GetBinPath(UmbracoObjectTypes objectType)
         {
-            var binPath = Constants.System.Root + ",";
+            var binPath = Constants.System.RootString + ",";
             switch (objectType)
             {
                 case UmbracoObjectTypes.Document:
-                    binPath += Constants.System.RecycleBinContent;
+                    binPath += Constants.System.RecycleBinContentString;
```

Comment:
This line could just be 
```
return Constants.System.RecycleBinContentPathPrefix; // "-1,-20"
```

Reply:
Constants.System.RecycleBinContentPathPrefix  has a "," in the end